lib/chef/provider/azure_resource_template.rb

```diff
@@ -33,8 +33,8 @@ def deployment
       end
 
       def template
-        template_src_file = ::File.join(Chef::Config[:chef_repo_path], new_resource.template_source)
-        fail "Cannot find file: #{template_src_file}" unless ::File.file?(template_src_file)
+        template_src_file = new_resource.template_source
```

Comment:
Please fix this breaking change by checking if the given path is absolute, and revert to the old behaviour if it is not:
``` 
if (Pathname.new(new_resource.template_source)).absolute? then 
  new_resource.template_source
else
  ::File.join(Chef::Config[:chef_repo_path]
end
  ```